config.json

```diff
@@ -46,7 +44,7 @@
       "mesosphere/dcos/1.11/**",
       "mesosphere/dcos/1.12/**",
       "mesosphere/dcos/1.13/**",
-      "mesosphere/dcos/2,1/**"
+      "mesosphere/dcos/2.1/**"
```

Comment:
why is 2.0 not specified here or anywhere else in this page?

Reply:
in my understanding the purpose of listing versions under `DO_NOT_BUILD` is to speed up builds. `2.0` likely could be added in all of those branches. on the other hand some of the branches (and respective pipelines) and the reference to `1.14` are known to be obsolete, so this file could need some attention in general. 

also "mesosphere/**" might be an easier glob for ksphere's configs.

Reply:
This is right Pierre. We need to add 2.0 only to the other product lists. But that's not the intent of this PR.